src/main/java/Todo.java

```diff
@@ -0,0 +1,14 @@
+public class Todo extends Task {
+
+    public Todo(String description) {
+        this.description = description;
+        this.status = status;
+        this.alphabet = 'T';
```

Comment:
you may want to change the name "alphabet" to  eg. "taskType" to make it clearer